src/zenml/utils/materializer_utils.py

```diff
@@ -102,7 +95,7 @@ def load_model_from_metadata(model_uri: str) -> Any:
     return model
 
 
-def model_from_model_artifact(model_artifact: ModelArtifact) -> Any:
+def model_from_model_artifact(model_artifact: ArtifactResponseModel) -> Any:
```

Comment:
Should this check that the artifact type is actually `model` in this case? Otherwise it seems like it's just a `load_artifact` function